CircleciScripts/upload_xcframeworks.py

```diff
@@ -0,0 +1,25 @@
+import boto3
+from botocore.exceptions import ClientError
+from framework_list import xcframeworks
+from functions import log
+
+def upload_file(file_name, bucket, object_name): 
+    s3_client = boto3.client('s3')
+    try:
+        s3_client.upload_file(file_name, bucket, object_name)
+    except ClientError as e:
+        logging.error(e)
+        sys.exit(e.response['Error']['Code'])
+    
+        
+project_dir = os.getcwd()
+archive_path = f"{project_dir}/xcframeworks/output/archives"
+
+bucket = sys.argv[1]
+version = sys.argv[2]
+
```

Comment:
As above, I suggest explicit validations for bucket & version.

Reply:
Added validation check